inc/saihostintf.h

```diff
@@ -186,7 +191,7 @@ typedef enum _sai_hostif_trap_type_t
     /** default action is forward */
     SAI_HOSTIF_TRAP_TYPE_IGMP_TYPE_V3_REPORT = 0x000000009,
 
-    /** default action is trap */
```

Comment:
I think for this, there is a prerequisite that the sampling is enabled. If not enabled this trap has no significance. Once the sampling is enabled, the default should be trap.